search_service/proxy/atlas.py

```diff
@@ -256,18 +255,13 @@ def fetch_table_search_results_with_filter(self, *,
         if table_filter_value:
             filters.append(('name', 'CONTAINS', table_filter_value[0]))
 
-        classifications: List[str] = list()  # noqa: E701
-
-        if badges_filter_value or tags_filter_value:
-            classifications = list(set(badges_filter_value + tags_filter_value))
-
         # Currently Atlas doesn't allow mixing search by filters and classifications
         if filters:
             query_params = self._prepare_basic_search_query(self.page_size, page_index,
                                                             filters=filters)
-        elif classifications:
+        elif tags_filter_value:
             query_params = self._prepare_basic_search_query(self.page_size, page_index,
-                                                            classifications=classifications)
+                                                            classification=tags_filter_value[0])
```

Comment:
@mgorsk1  so `tags_filter_value` is a list coming from frontend service ? Can we filter based on multiple tags at the same time ?

Reply:
that's what I was hoping for but unfortunately no - the only way would be to make several queries and join results, but that would mess up interpretation of popularityScore

Reply:
looks good to me.